src/Tickets/Commerce/Hooks.php

```diff
@@ -394,4 +397,26 @@ public function filter_js_include_checkout_url( $urls ) {
 
 		return $urls;
 	}
+
+	/**
+	 * Add a post display state for special Event Tickets pages in the page list table.
+	 *
+	 * @since TBD
+	 *
+	 * @param array   $post_states An array of post display states.
+	 * @param WP_Post $post        The current post object.
+	 */
+	public function add_display_post_states( $post_states, $post ) {
+
+		if ( tribe( Checkout::class )->get_page_id() === $post->ID ) {
+			$post_states['tec_tickets_commerce_page_checkout'] = __( 'Tickets Commerce Checkout Page', 'event-tickets' );
+		}
+
+		if ( tribe( Success::class )->get_page_id() === $post->ID ) {
+			$post_states['tec_tickets_commerce_page_success'] = __( 'Tickets Commerce Success Page', 'event-tickets' );
+		}
```

Comment:
Move this one into a method in the actual Success Class and call it here, so we dont have any logic on on Hooks class.